apps/site/pages/en/learn/typescript/run-natively.md

```diff
@@ -36,6 +55,10 @@ At the time of writing, the experimental support for TypeScript in Node.js has s
 
 You can get more information on the [API docs](https://nodejs.org/docs/latest/api/typescript.html#typescript-features).
 
+### Configuration
+
+Node.js's TypeScript loader ([Amaro](https://github.com/nodejs/amaro)) does not check the `tsconfig.json` file when transforming TypeScript code. In order to configure TypeScript to recognize (and match) this behavior when type checking, we recommend setting the `compilerOptions` listed in [here](https://nodejs.org/api/typescript.html#type-stripping), as well as using TypeScript on version **5.7 or higher**.
```

Comment:
Nit, but I don't really like the phrase "does not check", just becuase that implies to me that Amaro is "wrong" in some way, but I don't have a better idea.